api/v1alpha1/amaltheasession_types.go

```diff
@@ -161,6 +162,11 @@ type Ingress struct {
 	// +optional
 	// The name of the TLS secret, same as what is specified in a regular Kubernetes Ingress.
 	TLSSecret *SessionSecretRef `json:"tlsSecret,omitempty"`
+	// +optional
+	// The path prefix that will be used in the ingress. If left unset or set to "" then
```

Comment:
What is the default value when it isn't set? I'm not sure if it can be `nil` since it's not a pointer. If it can be `nil` then you should check against it as well in the above file code: `if cr.Spec.Ingress.PathPrefix == "" {`; if not, then please change this comment to say what's the default value.

Reply:
The default for a string when it is not set is `""` - just empty string. The default for a pointer that is not set is `nil`.

Reply:
But as @leafty suggested we can just set the default to `/` that works better.